### PR TITLE
Writing react plugins blog

### PR DESCRIPTION
**Pull request by peterholmberg**
Short blog post about the basics for creating a react panel.

### CODE REVIEW

**Comment by tomwilkie** [comment]
@jkld11 ming reviewing please?

**Comment by tomwilkie** [comment]
@peterholmberg we'd like to publish this on 26/03/2019 is thats okay?

**Comment by peterholmberg** [comment]
I was hoping that me and @daniellee could go through this since he has written many of the posts regarding plugins before we post. But can set aside some time for that before the 26th.

**Comment by tomwilkie** [comment]
Yes please, if possible.  Let me know if not.

File: content/blog/2019-02-26-writing-react-plugins.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  - "Grafana"
+---
+
+In this blog post we will go through how you can create plugins for Grafana using [ReactJS](https://reactjs.org). This
+presumes you have some basic knowledge about writing components in React.
+ 
+_(complete code for the example used in this post can be found [here](https://github.com/grafana/react-plugin-examples/tree/master/rss-panel))._
+
+In Grafana 6.0 we're starting the migration to using React in Grafana. This allows you to write plugins
+using React instead of Angularjs. To make writing plugins easier while also exposing some of the components used in
```
Comment (daniellee) [inline content/blog/2019-02-26-writing-react-plugins.md]: AngularJS

File: content/blog/2019-02-26-writing-react-plugins.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+---
+
+In this blog post we will go through how you can create plugins for Grafana using [ReactJS](https://reactjs.org). This
+presumes you have some basic knowledge about writing components in React.
+ 
+_(complete code for the example used in this post can be found [here](https://github.com/grafana/react-plugin-examples/tree/master/rss-panel))._
+
+In Grafana 6.0 we're starting the migration to using React in Grafana. This allows you to write plugins
+using React instead of Angularjs. To make writing plugins easier while also exposing some of the components used in 
+Grafana to plugins, we're launching the [@grafana/ui](https://www.npmjs.com/package/@grafana/ui) npm package.
```
Comment (daniellee) [inline content/blog/2019-02-26-writing-react-plugins.md]: This sentence is a bit clunky. How about:

We are making it easier to write React plugins by releasing a Grafana component library - the new [@grafana/ui](https://www.npmjs.com/package/@grafana/ui) npm package.

File: content/blog/2019-02-26-writing-react-plugins.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+Now we're ready to move on!
+
+## The Structure
+
+Grafana needs some basic project structure in your plugin. Grafana will look for a `plugin.json` located in a `src`
+directory. The `plugin.json` should contain some information about your plugin; you can read more about it
+[here](http://docs.grafana.org/plugins/developing/plugin.json/).
+
+Also within the `src` directory we need a `module.tsx` file. In this file, we will introduce the first magic from our
```
Comment (daniellee) [inline content/blog/2019-02-26-writing-react-plugins.md]: Nice - like the use of `magic`.

File: content/blog/2019-02-26-writing-react-plugins.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+## Types
+
+We strongly encourage you to use types in your panel. This makes it easier for you and others to spot potential bugs. In
+this example we've added some types for `RssFeed`, `RssFeedItem`, and `RssOptions`. These are located in `src/types.ts`.
+
+## Building
+
+Grafana expects a built javascript module for running the plugin. We're using webpack in our Rss-plugin example.
```
Comment (daniellee) [inline content/blog/2019-02-26-writing-react-plugins.md]: Would adjust this sentence. Maybe something like: 

To be able to load the plugin, Grafana expects the code to be in plain JavaScript. We're using webpack for the build step to transpile TypeScript to JavaScript in our Rss-plugin example.

File: content/blog/2019-02-26-writing-react-plugins.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+## Testing
+
+Start your grafana-server again, and make sure that your plugin is registered.
+
+`Registering plugin logger=plugins name="Rss Panel"`
+
+Add a new panel to a dashboard, and locate your new panel in the visualization picker.
```
Comment (daniellee) [inline content/blog/2019-02-26-writing-react-plugins.md]: Can you add an image of the finished panel?

**Pull request merged**